volcanoes/map.conf

```
@@ -0,0 +1,13 @@
+team.2.color = blue
+team.2.pos = (15,-63,-97)
+team.1.pos = (16,-63,101)
+team.1.color = red
+team.1 = red
+author = Oliver4
+rotation = z
+h = 140
+team.2 = blue
+name = Two Volcanoes
+initial_stuff = default:pick_steel,default:torch 10,shooter_guns:pistol
```

This gives an unloaded pistol,
```suggestion
initial_stuff = default:pick_steel,default:torch 10,shooter_guns:pistol_loaded
```